src/infosquito/es_crawler.clj

```diff
@@ -20,7 +20,7 @@
   (let [res (esd/search es index (name item-type)
               :query       (q/match-all)
               :_source     ["_id"]
-              :sort        ["_doc"]
+              :sort        ["id"]
```

Comment:
Too bad you can't sort on `_id`. Though, that would not affect performance.
I've checked the mappings for the `id` field, and they are all keyword. Default mapping settings for this field are already sufficient at this time.